Makefile

```diff
@@ -11,6 +11,9 @@
 GO=CGO_ENABLED=0 go
 GOCGO=CGO_ENABLED=1 go
 
+DBPROVIDER ?= github.com/edgexfoundry/edgex-go/internal/pkg/db/mongo
```

Comment:
How would you support more than one DB provider at a time? For example in the case where a customer wanted Redis for core-data and Mongo for everything else? Additional vars in the makefile?

Reply:
Yes, that is one option, another is reading a configuration & setting stuff up during build time. That would be necessary in case the db-provider needs some additional changes/resources during the build process, such as a docker-compose fragment if the DB is not embedded but needs a server running. 

I've not included any solution to this in the PR exactly because it would be too opinionated and would side-track from the goal of the PR, i.e. import & initialize the chosen database.